src/System.Data.SqlClient/tests/ManualTests/SQL/ParameterTest/OutputParameter.cs

```diff
@@ -0,0 +1,54 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Data;
+using System.Data.SqlClient;
+using System.Data.SqlTypes;
+using System.Diagnostics;
+
+namespace System.Data.SqlClient.ManualTesting.Tests
+{
+    public static class OutputParameter
+    {
+        public static void Run(string connectionString)
+        {
+            Console.WriteLine("Starting 'OutputParameter' tests");
+            InvalidValueInOutParam(connectionString);
+        }
+
+        // Dev11 Bug #336181: Regression: Changing the value of an output parameter to a value of different type throws System.FormatException
```

Comment:
I suggest removing old internal bug numbers in public code. They can be found in desktop code if we ever need them.